backend/apid/graphql/event_filter.go

```diff
@@ -0,0 +1,46 @@
+package graphql
+
+import (
+	v2 "github.com/sensu/sensu-go/api/core/v2"
+	"github.com/sensu/sensu-go/backend/apid/graphql/globalid"
+	"github.com/sensu/sensu-go/backend/apid/graphql/schema"
+	"github.com/sensu/sensu-go/graphql"
+	"github.com/sensu/sensu-go/types"
+	"github.com/sensu/sensu-go/util/strings"
+)
+
+var _ schema.EventFilterFieldResolvers = (*eventFilterImpl)(nil)
+
+//
+// Implement EventFilterFieldResolvers
+//
+
+type eventFilterImpl struct {
+	schema.EventFilterAliases
+}
+
+// ID implements response to request for 'id' field
+func (*eventFilterImpl) ID(p graphql.ResolveParams) (string, error) {
+	return globalid.EventFilterTranslator.EncodeToString(p.Source), nil
+}
+
+// IsTypeOf is used to determine if a given value is associated with the type
+func (*eventFilterImpl) IsTypeOf(s interface{}, p graphql.IsTypeOfParams) bool {
+	_, ok := s.(*types.EventFilter)
```

Comment:
It would be preferable to use `*v2.EventFilter` instead. That way we can avoid adding another alias to the `types` package.